sphinx/ext/napoleon/docstring.py

```diff
@@ -260,14 +260,18 @@ def _consume_field(self, parse_type: bool = True, prefer_type: bool = False
         _descs = self.__class__(_descs, self._config).lines()
         return _name, _type, _descs
 
-    def _consume_fields(self, parse_type: bool = True, prefer_type: bool = False
-                        ) -> List[Tuple[str, str, List[str]]]:
+    def _consume_fields(self, parse_type: bool = True, prefer_type: bool = False,
+                        multiple: bool = False) -> List[Tuple[str, str, List[str]]]:
         self._consume_empty()
         fields = []
         while not self._is_section_break():
             _name, _type, _desc = self._consume_field(parse_type, prefer_type)
-            if _name or _type or _desc:
-                fields.append((_name, _type, _desc,))
+            if multiple and _name:
+                for name in _name.split(","):
+                    fields.append((name.strip(), _type, _desc))
+            else:
+                if _name or _type or _desc:
+                    fields.append((_name, _type, _desc,))
```

Comment:
since you don't have any other clauses, this is equivalent to:
```suggestion
            elif _name or _type or _desc:
                fields.append((_name, _type, _desc,))
```

Reply:
Fixed.